src/devices/src/virtio/vsock/unix/muxer.rs

```diff
@@ -1305,4 +1320,63 @@ mod tests {
 
         assert!(!ctx.muxer.has_pending_rx());
     }
+
+    #[test]
+    fn test_regression_handshake() {
+        // Address one of the issues found while fixing the following issue:
+        // https://github.com/firecracker-microvm/firecracker/pull/1910
```

Comment:
Hmm, actually this links to a PR, not issue; same for the next test.

Reply:
Pasted the wrong link :)

Fixed now.